communication/slack-config/sig-release/config.yaml

```diff
@@ -5,3 +5,6 @@ channels:
   - name: release-management
     id: CJH2GBF7Y
   - name: release-notes
+  - name: release-bug-triage
+  - name: release-docs
+  - name: release-emeritus
```

Comment:
Remove this. The EA is a single person and should be working mostly in #sig-release.